Sophos EDR: documentation (#296)
SEKOIA-IO/documentation: docs/operation_center/integration_catalog/endpoint/sophos_edr.md

uuid: 3f330d19-fdea-48ac-96bd-91a447bb26bd
name: Sophos EDR
type: intake

## Overview

> Sophos EDR, monitor, detects and mitigates threats on endpoints.

Reviewer: ```suggestion
Sophos EDR monitors, detects and mitigates threats on endpoints.
```

This EDR reduces the attack surface and prevent attacks from running with an anti-exploit, an anti-ransomware and advanced control technology.

> This setup guide shows how to forward events produced by Sophos EDR to SEKOIA.IO.

Reviewer 2: ```suggestion
This setup guide describe how to forward events produced by Sophos EDR to SEKOIA.IO.
```

{!operation_center/integration_catalog/generated/sophos-edr_do_not_edit_manually.md!}

## Configure

### Credentials

> In the Sophos Central Admin, go to `Global Settings > API Credentials Management`.

Reviewer 2: ```suggestion
In the Sophos Central Admin console, go to `Global Settings > API Credentials Management`.
```

> Create a new token with `Add Credential`. Type a Credential name, select the appropriate role and add a description.

Reviewer 2: ```suggestion
Create a new token with `Add Credential`. Complete the name, appropriate role and description.
```

On the `API credential Summary`, copy the `Client ID` and the `Client Secret`.

### Create the intake

Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the format `AWS CloudTrail`.

### Pull events

> Get the [Sophos Central SIEM integration script](https://github.com/sophos/Sophos-Central-SIEM-Integration). Configure it according [this guide](https://support.sophos.com/support/s/article/KB-000036372?language=en_US)

Reviewer 2: ```suggestion
Get the [Sophos Central SIEM integration script](https://github.com/sophos/Sophos-Central-SIEM-Integration) and configure it by following [this Sophos guide](https://support.sophos.com/support/s/article/KB-000036372?language=en_US).
```

### Forward events

> See our [Ingestion methods page](https://docs.sekoia.io/operation_center/data_collection/ingestion_methods/) to select the appropriate methods to forward your events to sekoia.io according to your needs.

Reviewer 2: ```suggestion
See the [Ingestion methods page](https://docs.sekoia.io/operation_center/data_collection/ingestion_methods/) to select the appropriate method to forward your events to SEKOIA.IO according to your needs.
```

> Don’t use "I", "our", "us", "we", use "you".

https://zeroheight.com/6104b1d12/p/859e67-pronouns

> When referring to SEKOIA.IO, always put the name in capital letters. 

https://zeroheight.com/6104b1d12/p/9392ba-capitalization
